lib/controllers/register.js

```diff
@@ -89,6 +89,20 @@ function applyDefaultAccountFields(config,req) {
     req.body.surname = 'Anonymous';
   }
 }
+/**
```

Comment:
FWIW: Initially I imagined us having a `account-defaults.json` file, or something similar, which basically gets extended with `req.body` data to supplement in any missing (but optional) fields.